norse/torch/functional/lif.py

```diff
@@ -22,13 +22,24 @@ class LIFParameters(NamedTuple):
         alpha (float): hyper parameter to use in surrogate gradient computation
     """
 
-    tau_syn_inv: torch.Tensor = torch.as_tensor(1.0 / 5e-3)
-    tau_mem_inv: torch.Tensor = torch.as_tensor(1.0 / 1e-2)
-    v_leak: torch.Tensor = torch.as_tensor(0.0)
-    v_th: torch.Tensor = torch.as_tensor(1.0)
-    v_reset: torch.Tensor = torch.as_tensor(0.0)
-    method: str = "super"
-    alpha: float = torch.as_tensor(100.0)
+    def __init__(
+        self,
+        tau_syn_inv: torch.Tensor = torch.as_tensor(1.0 / 5e-3),
+        tau_mem_inv: torch.Tensor = torch.as_tensor(1.0 / 1e-2),
+        v_leak: torch.Tensor = torch.as_tensor(0.0),
+        v_th: torch.Tensor = torch.as_tensor(1.0),
+        v_reset: torch.Tensor = torch.as_tensor(0.0),
+        method: str = "super",
+        alpha: float = torch.as_tensor(100.0),
+    ):
+        super().__init__()
+        self.tau_syn_inv = tau_syn_inv
+        self.tau_mem_inv = tau_mem_inv
+        self.v_leak = v_leak
+        self.v_th = v_th
+        self.v_reset = v_reset
+        self.method = method
+        self.alpha = alpha
```

Comment:
I thought in this changeset we wouldn't switch over to torch.nn.Module for the parameters.

Reply:
Indeed, this is a remnant of the previous example. Thanks for pointing this out!